### PR TITLE
init conversion geo enum to geojson enum

### PR DESCRIPTION
**Pull request by datanel**
convert geo::types::* to geojson::geometry::Value

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -18,4 +18,6 @@ with-serde = ["serde","serde_json"]
 [dependencies]
 rustc-serialize = { version = "0.3", optional = true }
 serde = { version = "~0.8", optional = true }
-serde_json = { version = "~0.8", optional = true }
\ No newline at end of file
+serde_json = { version = "~0.8", optional = true }
+geo = "0.3.1"
```
Comment (azime) [inline Cargo.toml]: geo = "^0.3"

File: src/conversion.rs [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    ($x: expr, $y: expr, $epsilon: expr) => {
+        let a = $x.abs();
+        let b = $y.abs();
+        let delta = (a - b).abs();
+
+        if a.is_infinite() ||
+            a.is_nan() ||
+            b.is_infinite() ||
+            b.is_nan() {
+            panic!("Assertion failed: Non comparable value ({} = {}, {} = {})",  stringify!($x), $x, stringify!($y), $y);
```
Comment (azime) [inline src/conversion.rs]: line length

File: src/conversion.rs [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+fn create_point_type<T>(point: geo::Point<T>) -> PointType
+    where T: Float
+{
+    let x: f64 = point.x().to_f64().unwrap();
+    let y: f64 = point.y().to_f64().unwrap();
+
+    vec![x, y]
+}
+
+fn create_line_string_type<T>(line_string: &geo::LineString<T>) -> LineStringType
```
Comment (antoine-de) [inline src/conversion.rs]: not sure, because it's a utiliy function only used in this file no ?

File: src/conversion.rs [hunk 4]
```diff
@@ -0,0 +1,402 @@
...
+                    stringify!($x), $x, stringify!($y), $y);
+        } else if a.is_zero() || b.is_zero() {
+            if delta > $epsilon {
+                panic!("Assertion failed: ({} = {}, {} = {}, delta = {})",
+                    stringify!($x), $x, stringify!($y), $y, delta / b);
+            }
+        } else {
+            let normalized_delta = delta / b;
+            if normalized_delta > $epsilon {
+                panic!("Assertion fai116.34led: ({} = {}, {} = {}, delta = {})",
```
Comment (pbougue) [inline src/conversion.rs]: Copy typo here

**Pull request merged**